skyportal/handlers/api/followup_request.py

```diff
@@ -497,14 +521,21 @@ def delete(self, request_id):
         if not api.implements()['delete']:
             return self.error('Cannot delete requests on this instrument.')
 
-        api.delete(followup_request)
+        response = api.delete(followup_request)
 
-        obj_key = followup_request.obj.internal_key
+        transaction = FacilityTransaction(
+            request=http.serialize_requests_request(response.request),
+            response=http.serialize_requests_response(response),
+            followup_request=followup_request,
+            initiator=self.associated_user_object,
+        )
 
-        DBSession().delete(followup_request)
+        DBSession().add(transaction)
+        DBSession().add(followup_request)
```

Comment:
Again, I don't think this is necessary. 
e.g.
``` python
In [2]: u = User.query.get(4)                                                                                                                
                                                                                                                                              
In [3]: u                                                                                                                                    
Out[3]: <User(id=4, created_at=2020-09-12 16:45:55.210345, modified=2020-09-12 16:45:55.210345, username=groupadmin@cesium-ml.org, first_name=
None, last_name=None, contact_email=None, contact_phone=None, oauth_uid=None, preferences=None)>                                              
                                                                                                                                              
In [4]: u.preferences = {"ab": 123}                                                                                                          
                                                                                                                                              
In [5]: DBSession().commit()                                                                                                                  
                                                                                                                                              
In [6]: u = User.query.get(4)                                                                                                                
                                                                                                                                              
In [7]: u.preferences                                                                                                                        
Out[7]: {'ab': 123}                                                                                                                           
```

Reply:
fixed